internal/kms/kes.go

```diff
@@ -138,9 +139,30 @@ func NewWithConfig(config Config) (KMS, error) {
 			}
 		}
 	}()
+
+	go func() {
+		c.keepKeyInCache()
+	}()
 	return c, nil
 }
 
+// Request KES keep an up-to-date copy of the KMS master key to allow minio to start up even if KMS is down. The
+// cached key may still be evicted if the period of this function is longer than that of KES .cache.expiry.unused
+func (c *kesClient) keepKeyInCache() {
+	ctx := context.Background()
+
+	ticker := time.NewTicker(10 * time.Second)
+	defer ticker.Stop()
+	for {
+		select {
+		case <-ticker.C:
+		case <-ctx.Done():
+			return
+		}
+		_ = c.ValidateKey(ctx)
```

Comment:
```suggestion
		_ = c.ValidateKey(ctx)
```

what if ValidateKey returns an error result? who will monitor this?

Reply:
It's OK if it returns a result - the point of this code is to refresh the KES cache if it is refreshable i.e. if KES is available. If it fails then KES is unavailable and it does not matter if this code fails. 
I will make this function return a bool and simply invoke it within the timer.

Reply:
As you mention below - I will add a loggerOnce to this function

Reply:
Implemented